src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 0.3s;
```

Comment:
You've used value **_0.3s_** several times, so it is a sign, to create a variable